.build/automation/variables.yml

```diff
@@ -0,0 +1,29 @@
+variables:
+- group: Xamarin Release
+- group: XQA Secrets
+- group: Xamarin-Secrets
+- group: xamops-azdev-secrets
+
+- name: DefaultBuildConfiguration
+  value: Release
+
+- name: isMainBranch
+  value: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
+- name: isLocBranch
+  value: $[eq(variables['Build.SourceBranch'], 'refs/heads/loc')]
+- name: isTargetMainBranch
+  value: $[eq(variables['System.PullRequest.TargetBranch'], 'refs/heads/main')]
+- name: isTargetLocBranch
+  value: $[eq(variables['System.PullRequest.TargetBranch'], 'refs/heads/loc')]
+- name: isLocPRBranch
+  value: $[startsWith(variables['System.PullRequest.SourceBranch'], 'refs/heads/$(LocBranchPrefix)')]  
```

Comment:
I tried using `$(...)` to reference variables from a new variable declaration and it didn't work. Looks like you should use `variables['...']`, but it maybe worth double checking

Reply:
This working correctly in UITools, so I am going to leave it as is.

Reply:
didn't work for me either